docs/source/user/api_change.rst

```diff
@@ -9,7 +9,105 @@ The changes are tabulated according to the module input file, line number, and f
 The line number corresponds to the resulting line number after all changes are implemented.
 Thus, be sure to implement each in order so that subsequent line numbers are correct.
 
-OpenFAST v3.5.4 to OpenFAST v3.5.5 
+
+
+OpenFAST v3.5.5 to OpenFAST 4.0.0
+---------------------------------
+
+The HydroDyn module was split into HydroDyn and SeaState.  This results in a
+completely new input file for SeaState, and complete revision of the HydroDyn
+input file.  See examples in the regression tests for the new formats.
+
+New modules AeroDisk (see :numref:`ADsk`) and Simplified-ElastoDyn (see :numref:`SED`)
+were added.  See documentation on those modules for exmple input files.
+
+============================================= ======= ==================== ========================================================================================================================================================================================================
+Modified in OpenFAST `dev`
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Module                                        Line    Flag Name            Example Value
+============================================= ======= ==================== ========================================================================================================================================================================================================
+OpenFAST                                      15      CompAero\**          2   CompAero        - Compute aerodynamic loads (switch) {0=None; 1=AeroDisk; 2=AeroDyn; 3=ExtLoads}
+OpenFAST                                      13      CompElast            3   CompElast       - Compute structural dynamics (switch) {1=ElastoDyn; 2=ElastoDyn + BeamDyn for blades; 3=Simplified ElastoDyn}
```

Comment:
Isn't there also a new CompSeaState switch?

Reply:
Yes.  There should be some additional lines here.  I'll add them.

Reply:
Updated in [e973b76](https://github.com/OpenFAST/openfast/pull/2586/commits/e973b76cf68077910f974fb03a2c963c6f49cbad)